src/ChainSafe.GamingSDK.EVM/Web3/Core/Analytics/ApiAnalyticsClient.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using ChainSafe.GamingWeb3.Environment;
+
+namespace ChainSafe.GamingWeb3.Analytics
+{
+    public class ApiAnalyticsClient : IAnalyticsClient
+    {
+        private const string LoggingUrl = "https://game-api-stg.chainsafe.io/logging/logEvent";
```

Comment:
Is it possible not to hardcode any URLs but read them from settings, configs, or environment?

Reply:
If we're going to allow on-prem installation of the API server, then it definitely makes sense to make the API endpoint configurable. It can live next to the other configurations (chain ID etc.)

Reply:
This complicates the analytics feature though, since to report the data to the correct backend, the events should be sent to our instance of the API server. And if that's the only API endpoint we're using in version 2, then maybe it doesn't make sense to make it configurable?

Reply:
Yeah, this is a good one. So far I see two reasons to make it configurable: the first reason is to separate staging and production environments because we will find ourselves in a mess of `Project IDs` that do not exist on pord while reporting analytics from stage; the second reason is to prepare for the self-hosted solution.

So I see there two solutions:
1. Make it configurable and make putting an empty URL in settings an act of opting out of the analytics collection.
2. Ask API to deal with both sets of Project IDs on the analytics endpoint and introduce a flag to opt-out of the analytics collection.

Reply:
I've created a new issue for this: #468